azext_iot/tests/test_iot_central_int.py

```diff
@@ -267,6 +274,45 @@ def test_central_api_token_methods_CRD(self):
             }
             assert token_info_basic in token_list
 
+    @pytest.mark.skipif(not STORAGE_CS, reason="empty azext_iot_central_sa_cs env var")
```

Comment:
Since we are adding a bunch of IT's and this file is growing to ~500lines now, should we think about splitting the IT classes up a bit more?

IoTCentralDeviceAndDeviceTemplateIntegrationTests
IoTCentralUserTests
IoTCentralCDETests
IoTCentral...

Reply:
this one is optional, only do this if you think its easier to debug issues when splitting the tests up by use case